asdf_astropy/converters/coordinates/tests/test_representation.py

```diff
@@ -6,8 +6,13 @@
 
 from asdf_astropy.testing.helpers import assert_representation_equal
 
+# https://github.com/astropy/astropy/pull/14763 moved Geodetic representations from
+# astropy.coordinates.earth to astropy.representations.geodetic we do not have
+# converters for these types so we skip the geodetic classes in these tests
```

Comment:
The `astropy.coordinates.representations.geodetic` objects seem to be subclasses of the same `BaseRepresentation` object, so they should already be serializable using the existing converter. Please explain why we need to ignore these? The only issue I see is that `asdf-coordinates-schemas` does not list the new geodetic types.

If this is necessary, please either add converter support for these objects or open an explicit issue to track the creation of converters for these objects.

Reply:
I opened an issue for this.

https://github.com/astropy/asdf-astropy/issues/182

I don't see any converter that lists `BaseRepresentation`. Which converter should be supporting these new classes?

Reply:
No, those are just grouping classes only, so that one can do `isinstance` checks.

Reply:
As you pointed out in https://github.com/asdf-format/asdf-coordinates-schemas/pull/40#pullrequestreview-1440302867
additional updates will be required to add support for the Geodetic representations. I removed the skip for the tests against these classes from this PR.

This PR should now address class path changes introduced in https://github.com/astropy/astropy/pull/14792